.github/workflows/deploy.yml

```diff
@@ -76,7 +76,7 @@ jobs:
           cf_space:    ${{secrets.CF_SPACE }}
           cf_username: ${{secrets.CF_USER}}
           cf_password: ${{secrets.CF_PASSWORD}}
-          command: push funding-service-design-authenticator-dev -f manifest-dev.yml --var RSA256_PRIVATE_KEY_BASE64="${{secrets.RSA256_PRIVATE_KEY_BASE64}}" --var RSA256_PUBLIC_KEY_BASE64="${{secrets.RSA256_PUBLIC_KEY_BASE64}}"  --var AZURE_AD_CLIENT_SECRET="${{secrets.AZURE_AD_CLIENT_SECRET}}" --var SECRET_KEY="${{secrets.SECRET_KEY}}" --var SESSION_COOKIE_NAME="${{secrets.SESSION_COOKIE_NAME}}"
+          command: push funding-service-design-authenticator-dev -f manifest-dev.yml --var RSA256_PRIVATE_KEY_BASE64="${{secrets.RSA256_PRIVATE_KEY_BASE64}}" --var RSA256_PUBLIC_KEY_BASE64="${{secrets.RSA256_PUBLIC_KEY_BASE64}}"  --var AZURE_AD_CLIENT_ID="${{secrets.AZURE_AD_CLIENT_ID}}"  --var AZURE_AD_CLIENT_SECRET="${{secrets.AZURE_AD_CLIENT_SECRET}}" --var SECRET_KEY="${{secrets.SECRET_KEY}}" --var SESSION_COOKIE_NAME="${{secrets.SESSION_COOKIE_NAME}}"
```

Comment:
Did you set these values on the dev environment secrets?

Reply:
Ah just come back to this - yes getting ahead of myself have reverted on dev

Reply:
On review it seems better to just set it (as we need it) to get both from the github env vars and just set them as secrets now. Only development and unit_test envs now use the original AD client details so that it can pass testing although these tests should also better perhaps mock the Azure response.